HOWTO.md

```diff
@@ -523,10 +524,41 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    `/etc/fdo/manufacturing-serverr.conf.d/`. The paths will be checked in that
    same order.
 
-3. Execute `fdo-manufacturing-server` or run it as a service, see sample
-   file in
+3. Execute `fdo-manufacturing-server` using below command:
+   ```
+   sudo MANUFACTURING_SERVER_CONF=/usr/share/fdo/manufacturing_server.yml LOG_LEVEL=trace ./target/debug/fdo-manufacturing-server
```

Comment:
This command is very specific and conflicts with the pre-requisites of the HOWTO (tools built in release profile), and it also conflicts with step 2 of this section (default placement for the configuration file). 

You can add an specific section about environment variables and explain those there (`LOG_LEVEL`, `MANUFACTURING_SERVER_CONF`)

Reply:
added 'Environmental Variables' section .